product-service/constants/sql.ts

```diff
@@ -0,0 +1,4 @@
+export const PRODUCTS = 'SELECT products.id, products.title, products.description, products.price, stocks.count from products INNER JOIN stocks ON products.id=stocks.product_id';
```

Comment:
Why is it INNER JOIN but not LEFT JOIN? If there is no count specified for product you will not recieve the whole product.
I think it's better to get product with count = null instead of not get it at all.